soh/soh/SohMenuBar.cpp

```diff
@@ -557,6 +557,8 @@ void DrawEnhancementsMenu() {
                 UIWidgets::Tooltip("Nighttime Skulltulas will spawn during both day and night.");
                 UIWidgets::PaddedEnhancementCheckbox("Dampe Appears All Night", "gDampeAllNight", true, false);
                 UIWidgets::Tooltip("Makes Dampe appear anytime during the night, not just his usual working hours.");
+                UIWidgets::PaddedEnhancementCheckbox("Fisherman asks for fishing rod", "gFishingRodLeave", true, false);
```

Comment:
I feel like the menu item name should be a bit more concisely accurate, something like "Quit Fishing At Door" (though I'm not 100% on that one specifically; anyone else have a suggestion?). Also, needs proper capitalization, like the pattern for other item labels. Some got through in the past, but I'll probably make a PR to change that as I would like some uniformity there.

Reply:
I was stuck on including "Rod", but I like Quit Fishing At Door. I've changed it for now.